.clang-tidy

```diff
@@ -1,5 +1,5 @@
 ---
-Checks:          '-*,clang-diagnostic-*,clang-analyzer-*,-clang-analyzer-alpha*,google-*,-google-readability-todo,-google-runtime-references,-google-default-arguments,-clang-analyzer-core.uninitialized.UndefReturn,-clang-analyzer-core.UndefinedBinaryOperatorResult,-clang-analyzer-optin.cplusplus.VirtualCall'
+Checks:          '-*,clang-diagnostic-*,clang-analyzer-*,-clang-analyzer-alpha*,google-*,-google-readability-todo,-google-runtime-references,-google-default-arguments,-clang-analyzer-core.uninitialized.UndefReturn,-clang-analyzer-core.UndefinedBinaryOperatorResult,-clang-analyzer-optin.cplusplus.VirtualCall,-google-explicit-constructor'
```

Comment:
This tests you have excluded should be passing, why exclude it?

Reply:
This is intentional. We want to use universal initialization, using the "explicit" keyword disables that for those constructors.

Reply:
I am not familiar with that being the case. Can you point me to some documentation? `explicit` should just be keeping decay-able parameters from implicitly matching on the constructor.